src/style.css

```
@@ -1 +1,80 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+    
+.flexbox {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
```

where is box-shadow for header?
check design
```suggestion
.header {
  display: flex;
  justify-content: space-between;
  padding: 0 50px;
}
```
and name classes meaningful

---

flexbox name is ok when this class is used in many placed only for make flexbox from container